.changeset/sour-scissors-teach.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/dlc-btc-por-adapter': patch
```

Comment:
Should we make this a `minor`? Then this won't override the existing version image, giving us the option of reverting back if this causes any issues. 

```suggestion
'@chainlink/dlc-btc-por-adapter': minor
```